keyboards/fancytech/fancyalice66/fancyalice66.c

```diff
@@ -0,0 +1,40 @@
+/* Copyright 2023 Adophoxia <https://github.com/Adophoxia>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "quantum.h"
+
+#ifdef RGB_MATRIX_ENABLE
+bool rgb_matrix_indicators_kb(void) {
```

Comment:
```suggestion
bool rgb_matrix_indicators_kb(void) {
    if (!rgb_matrix_indicators_user() {
        return false;
    }
```

Reply:
Tried to do this a couple of weeks ago and I ran into compile issue where it acted that `rgb_matrix_indicator_user` didn't exist. Will test again when I get back to my PC to make sure it works fine.